Frontend/src/services/FlightService.js

```diff
@@ -32,38 +26,38 @@ const apiClient2 = axios.create({
 });
 
 export default {
-  userLogin(data) {
-    return apiClient.post("/users/login", data);
-  },
+  // userLogin(data) {
+  //   return apiClient.post("/users/login", data);
+  // },
   getFlights() {
-    return apiClient2.get("/tageswertung");
+    return apiClient2.get("tageswertung");
   },
   getFlight(flightId) {
-    return apiClient.get("/flights/" + flightId);
+    return apiClient.get("flights/" + flightId);
   },
   // These will be obsolete if the flight modell contains comments and description
   getComments() {
-    return apiClient2.get("/comments");
+    return apiClient2.get("comments");
   },
   getDescription() {
-    return apiClient2.get("/flightDescription");
+    return apiClient2.get("flightDescription");
```

Comment:
FlightDescription == report? Evtl müssen wir uns hier mal auf einen Namen einigen.

Reply:
ja, werde deinen Namen übernehmen, das ist nur noch so in der alten fake db die bei mir läuft. sobald wir die demo Daten haben wird die obsolet.